sdk/src/main/java/com/horizen/account/serialization/EthJsonMapper.java

```diff
@@ -0,0 +1,47 @@
+package com.horizen.account.serialization;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.horizen.evm.utils.BigIntegerDeserializer;
+import com.horizen.evm.utils.BigIntegerSerializer;
+
+import java.math.BigInteger;
+
+public class EthJsonMapper {
+
+    private static final ObjectMapper mapper;
+
+    static {
+        var module = new SimpleModule();
+        module.addSerializer(BigInteger.class, new BigIntegerSerializer());
+        module.addDeserializer(BigInteger.class, new BigIntegerDeserializer());
+        module.addSerializer(byte[].class, new EthByteSerializer());
+        module.addDeserializer(byte[].class, new EthByteDeserializer());
+        mapper = new ObjectMapper();
+        mapper.registerModule(module);
+        mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
So do we need to skip unknown RPC arguments or fail?
I assume before we used a policy `FAIL_ON_UNKNOWN_PROPERTIES`

Reply:
This was copied from the `ObjectMapper` that was used from `ApplicationJsonSerializer`. For reference, this is a working request on Goerli via Infura:
```
{
  "jsonrpc": "2.0",
  "id": 1,
  "method": "eth_call",
  "params": [
    {
      "to": "0x0000000000000000000000000000000000000002",
      "data": "0x516f2c0f",
      "foo": "bar"
    },
    "latest"
  ]
}
```
I think accepting unknown properties is an easy way to hide errors and would vote for removing it. We can still allow unknown properties on specific types via `@JsonIgnoreProperties(ignoreUnknown = true)`, e.g. on the `TransactionArgs` type to allow a request like the one avove.

Reply:
agree, let's fail